common/src/main/resources/META-INF/persistence.xml

```diff
@@ -91,7 +92,7 @@
         <class>org.apache.falcon.persistence.MonitoredEntityBean</class>
         <class>org.apache.falcon.persistence.PendingInstanceBean</class>
         <class>org.apache.falcon.persistence.EntitySLAAlertBean</class>
-
+        <class>org.apache.falcon.persistence.BacklogMetricBean</class>
```

Comment:
please add the class name every where currently its missing in mysql and derby in tag openjpa.MetaDataFactory.